libs/platform/experience/src/color/colors/colored/red.ts

```diff
@@ -2,6 +2,7 @@ import { Color } from '../../color.model';
 
 export const red: Color = {
   light: {
+    0: 'white',
```

Comment:
Is light red color white?

Reply:
position 0 is white for all non-bright colors.